notebooks/CubevizExample.ipynb

```diff
@@ -55,11 +51,12 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Here we download a JWST MIRI cube from [MAST](https://masttest.stsci.edu/) using `astroquery`, \n",
-    "to use as our example data. By default the downloaded file goes to your\n",
-    "temp directory, and thus may eventually be garbage collected (deleted) by your system.\n",
-    "If you would like to have the file permanently, simply uncomment the second line below\n",
-    "and provide the directory path on your system where you would like the file stored.\n",
+    "Now we load observations. If you already have the files on your local machine, you can \n",
+    "load them by passing their file paths to `load_data` as strings. For this example, \n",
+    "we will retrieve remote data from [MAST](https://masttest.stsci.edu/) via `astroquery`\n",
```

Comment:
Is it time to remove the "test", @havok2063 ?
```suggestion
    "we will retrieve remote data from [MAST](https://mast.stsci.edu/) via `astroquery`\n",
```

Reply:
Yes I think it's safe to remove the test here.